Lib/idlelib/query.py

```diff
@@ -220,6 +220,14 @@ def entry_ok(self):
             self.showerror("loader does not support get_filename",
                       parent=self)
             return None
+        except ImportError:
+            # Some special modules require this (e.g. os.path)
+            try:
+                file_path = spec.loader.get_filename()
+            except TypeError:
+                self.showerror("loader failed to get filename",
+                          parent=self)
```

Comment:
Delete erroneous argument, copied from previous error.
```suggestion
                self.showerror("loader failed to get filename.")
```

Reply:
showerror calls fixed.